### ISSUE TITLE
[Portal, Reauth] Suggest changing reauth page UI text to "Enter password to continue"

### ISSUE THREAD

**Issue by joyzoursky** [authgear/authgear-server#3146]
## Background
- Environment: Staging
- Test account: joyz+demo1@oursky.com
- Test project: pipe-flash-309 - https://pipe-flash-309.authgear-staging.com
- Reported by: QA tester

## Description
When user tries to do something that requires reauthentication, such as editing social login configuration, it will go to a "Enter password" page, and tester reports that they are "logged out" there, which means they can't understand that it is the reauthentication page.

Suggest rewording the page to use text like "Enter password to continue" to be clear.

## Steps to reproduce
1. Login to portal
2. Go to Authentication > Social / Enterprise Login
3. Click "Edit" button
4. See "Enter password" page, which is a bit confusing as it looks like logged out more than reauthenticating

## Authgear reauth screen
<img width="1552" alt="Screenshot 2023-06-20 at 12 12 03 PM" src="https://github.com/authgear/authgear-server/assets/12539486/ab77ce5c-45aa-4259-b27e-c6f0d7ff6834">

## GitHub reauth screen (ref)
![github](https://github.com/authgear/authgear-server/assets/12539486/5a01477c-8a6f-4928-9b16-3367e6910077)

**Comment by fungc-io** [comment]
I like the "Tips" under the github reauth page.

Maybe we can add a paragraph under the title too

> This page you are trying to visit requires that you re-enter your password.

**Comment by tung2744** [comment]
@fungc-io We don't have separated screen for auth / re-auth to enter password. If we want another ui text in the re-auth flow we need to separate them.
Also, `This page you are trying to visit requires that you re-enter your password.` is only suitable for re-auth to visit a page, but re-auth is not only for visiting a page.

**Comment by fungc-io** [comment]
Maybe change it to `This action you are trying to do requires that you re-enter your password.`

**Comment by tung2744** [comment]
@fungc-io Did you meant to add a paragraph at the bottom, or just update the title?

**Comment by tung2744** [comment]
Not sure if it is what you want:

<img width="711" alt="Screenshot 2023-06-29 at 1 21 53 PM" src="https://github.com/authgear/authgear-server/assets/34942210/80fc7f28-fe26-4f11-bb0e-5b24ae4d86ce">

**Comment by fungc-io** [comment]
How about between the title and the password field
In other pages we have some text in that area

**Comment by tung2744** [comment]
@fungc-io See if this is ok?

![Image](https://github.com/authgear/authgear-server/assets/34942210/7ae6a346-c6dc-4172-b430-806c193d322a)

**Comment by fungc-io** [comment]
😂 sorry i didnt know the title above is extremely similar

please change the text to 

zh-Hant:

> 繼續操作前須重新輸入密碼以認證你的身份

zh-Hans:

> 继续操作前须重新输入密码以认证你的身份

**Comment by fungc-io** [comment]
Tested on staging
